pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.3.1.RELEASE</version>
+        <version>1.5.5.RELEASE</version>
```

Comment:
Warum nicht 1.5.10? Damit das noch für die erste Story übrig bleibt?

Reply:
ja genau. Ich hatte auf die E-Mail geantwortet, aber die schafft es offenbar nicht in die Kommentare hier.